pkg/ecdsa/tss/message.go

```diff
@@ -40,13 +40,33 @@ func (m *AnnounceMessage) Type() string {
 }
 
 func RegisterUnmarshalers(broadcastChannel net.BroadcastChannel) {
-	broadcastChannel.RegisterUnmarshaler(func() net.TaggedUnmarshaler {
+	err := broadcastChannel.RegisterUnmarshaler(func() net.TaggedUnmarshaler {
```

Comment:
Having the recent `keep-core` version referenced after merging https://github.com/keep-network/keep-ecdsa/pull/535 we should update all those calls.